client/src/client-only-routes/ShowProjectLinks.js

```diff
@@ -173,23 +193,22 @@ const ShowProjectLinks = props => {
           solution={solution}
         />
       ) : null}
-      If you suspect that any of these projects violate the{' '}
-      <a
-        href='https://www.freecodecamp.org/news/academic-honesty-policy/'
-        target='_blank'
-        rel='noreferrer'
-      >
-        academic honesty policy
-      </a>
-      , please{' '}
-      <a
-        href={`/user/${username}/report-user`}
-        target='_blank'
-        rel='noreferrer'
-      >
-        report this to our team
-      </a>
-      .
+      <Trans i18nKey='certification.project.footnote'>
+        <a
+          href='https://www.freecodecamp.org/news/academic-honesty-policy/'
+          target='_blank'
+          rel='noreferrer'
+        >
+          placeholder
+        </a>
+        <a
+          href={`/user/${username}/report-user`}
+          target='_blank'
+          rel='noreferrer'
+        >
+          placeholder
```

Comment:
```suggestion
          report this to our team
```
Same principle as the previous placeholder.